internal/processor/stream_test.go

```diff
@@ -380,3 +380,46 @@ func TestInferredStream(t *testing.T) {
 		}
 	}
 }
+
+func TestDescribeToJson(t *testing.T) {
+	input := "Fields\n--------------------------------------------------------------------------------\nUSERID\tbigint\nFIRST_NAME\tstring\nLAST_NAME\tstring\nNICKNAMES\t" +
```

Comment:
It would be helpful if you could modify it to align with the format of other tests in the project, making it easier to add new tests.

Reply:
cool, what I meant was that you can enhance the code by incorporating the following structure, which allows to conveniently add test samples in the future:
``` go
tests := []struct {
		input   string
		want   string
		err string
	}{}
```

This approach simplifies the process of adding new test cases at a later stage.